app/controllers/users_controller.rb

```diff
@@ -8,17 +8,31 @@ def show
 
   def edit
     @user = current_user
-    @industries = User::PRIMARY_INDUSTRY
+    @industries = MentorIndustry.all
     @interests = User::TOP_3_INTERESTS
+    @career_stages = MentorIndustryUser.career_stages
+
+    MentorIndustry.all.each do |industry|
+      if !@user.mentor_industry_ids.include?(industry.id)
+        @user.mentor_industry_users.build(mentor_industry_id: industry.id)
+      end
+    end
   end
 
   def update
     @user = current_user
     if @user.update(user_params)
       redirect_to user_path(current_user)
     else
-      @industries = User::PRIMARY_INDUSTRY
+      @industries = MentorIndustry.all
       @interests = User::TOP_3_INTERESTS
+      @career_stages = MentorIndustryUser.career_stages
+
+      MentorIndustry.all.each do |industry|
+        if !@user.mentor_industry_ids.include?(industry.id)
+          @user.mentor_industry_users.build(mentor_industry_id: industry.id)
```

Comment:
parenthesis optional here

Reply:
The Ruby style guide claims we should leave these, from all that we could tell. Unless there are other instances in the code base we're trying to be consistent with.

Reply:
I don't believe these `mentee/mentor_mentorships` were here before I started working on this issue, but if I remember right, we changed how the `@user` instance variables were found when I was working with Ben.

Reply:
that's correct, they got added in the most recent PR merge